cpp/src/barretenberg/stdlib/recursion/verifier/verifier.test.cpp

```diff
@@ -136,7 +136,7 @@ template <typename OuterComposer> class stdlib_verifier : public testing::Test {
             verification_key_pt::from_witness(&outer_composer, verification_key_native);
 
         info("Constructing the ultra (inner) proof ...");
-        plonk::proof recursive_proof = prover.construct_proof();
```

Comment:
Name was vague. If anything, I'd call the outer proof, the one constructed by the recursive verifier, the recursive proof.